broadlink/__init__.py

```diff
@@ -90,12 +90,12 @@
     0x60C8: (lb1, "LB1", "Broadlink"),
     0x6112: (lb1, "LB1", "Broadlink"),
     0x2722: (S1C, "S2KIT", "Broadlink"),
+    0X4E2A: (tornado, "TOP SQ X", "Tornado"),
```

Comment:
We should consider naming the class with the model instead of the brand. Eventually Tornado will launch another device and we will have to create another class, and tornado_new is weird.

How about `top` or `sqx`? We need to make the type short so we can keep the code clean eg [here](https://github.com/home-assistant/core/blob/54425ae0f3c1e35c8040583edfe16ad3a1b7e243/homeassistant/components/broadlink/const.py#L8-L12).

Reply:
I have a "SMART-16X SQ (WIFI)" (the numbers change based on size). Reading through the [catalogue](https://www.tornado-top.co.il/catalog/) there's also a TOP SQ WIFI __X series (which at some point I assumed were identical). Based on the app description both should work, but that is untested. So I renamed the class to `xsq`, does that work?

Reply:
How does the model look like in the official app? If it looks specific (numbers included), the PIDs are different. If it looks generic, they are probably using the same PID for all these _X models.

Reply:
It's utterly generic, the most specific it gets is "Air-conditioner".

Reply:
@thewh1teagle What is your model?

Reply:
I think we should use `sq` or `sq1`.

Reply:
Sounds good, do you have a preference?

Reply:
I think `sq1` will look better if we need to create a `sq2` in the future. We can use `sq1` for the class, `SQ1` for the type and `SQ` for the model name.

Reply:
@felipediel Sorry for late, my model is ```SMART-12X SQ (WIFI)```

Reply:
Thank you!